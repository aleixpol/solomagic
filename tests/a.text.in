\mt sagu question ena'a / kenoa
\ma sagu           question   enaqa kenoa
\mb sa   -gu       question   enaqa keno -qang
\gl POSS -1SG.POSS P.question DEM   sea  -LOC
\ps poss -psuf     n          dem   n    -nsuf
