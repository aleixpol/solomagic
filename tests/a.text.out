\mt sagu question ena'a / kenoa
\ma sagu           question   ena'a kenoa
\mb sa   -gu       question   ena'a keno -'ang
\gl POSS -1SG.POSS P.question DEM   sea  -LOC
\ps poss -psuf     n          dem   n    -nsuf
